messages.lua

```diff
@@ -0,0 +1,185 @@
+local Messages = {}
+BuffDuty.Messages = Messages
+
+local setToDefault = "default"
+
+local defaultPublicTitle = "Dear $class$s, please support our raid with your buffs, love and care!"
+local defaultDutyLine = "Group$s $groups - {rt$i} $name {rt$i}"
+local defaultDutyWhisper = "Thank you $name, for supporting our raid today! Would you kindly attend to buffing group$s $groups when you are able."
+local defaultSingleTitle = "Looks like we only have one $class in the raid today. {rt1}$name{rt1}, dear, would you kindly provide everyone with your wonderful buffs!"
+local defaultSingleWhisper = "Dear $name, looks like you are the only $class in the raid today, would you kindly provide everyone with your wonderful buffs!"
+
+function Messages:Initialise()
+    -- Fixed messages
+    self.message_title = "(Buff Duty) • %s •"
+    -- Settable messages
+    self.public_title = defaultPublicTitle
+    self.duty_line = defaultDutyLine
+    self.duty_whisper = defaultDutyWhisper
+    self.single_title = defaultSingleTitle
+    self.single_whisper = defaultSingleWhisper
+end
+
+-- Validation functions return true = OK / false = Error and an error string
+local function validatePublicTitle(public_title)
+    if public_title == setToDefault then
+        return true
+    end
+    return true
+end
+Messages.validatePublicTitle = validatePublicTitle
+
+local function validateDutyLine(duty_line)
+    if duty_line == setToDefault then
+        return true
+    end
+    if not (string.find(duty_line, "$groups", 1, true) and string.find(duty_line, "$name", 1, true)) then
+        return false, "Duty Line format must contain: $groups and $name"
+    end
+    return true
```

Comment:
Maybe better raise exception?

Reply:
Haven't played with exceptions, or Assert, in Lua yet... could try it

Reply:
I've changed these to use `error("msg")` instead of `return false, "msg"`